examples/vertexattributes/vertexattributes.h

```diff
@@ -9,7 +9,7 @@
 #define TINYGLTF_IMPLEMENTATION
 #define STB_IMAGE_IMPLEMENTATION
 #define TINYGLTF_NO_STB_IMAGE_WRITE
-#define TINYGLTF_NO_STB_IMAGE
```

Comment:
Can you elaborate on this? It works fine for me with the define in place. Does this cause problems on iOS/MacOS?

Reply:
Good comment and catch here.  I originally made this change (also in gltfscenerendering.h) based on build failures on macOS and iOS.  However, I later made a change to how VulkanglTFModel.cpp is included in the build config for the xcode/examples project (now conditionally included in examples.h when tiny_gltf.h is not otherwise included or customized by an example).  This build config change seems to make the change here unnecessary.  It's been a while since I looked at this area of the code, but now after uncommenting `#define TINYGLTF_NO_STB_IMAGE` I can no longer duplicate build failures for the vertexattributes and gltfscenerendering examples on macOS and iOS.  I guess you can revert this change here and in gltfscenerendering.h